site/guide/state.md

```diff
@@ -179,9 +179,8 @@ use the following dependencies:
 [dependencies]
 rocket = "0.4.0-dev"
 diesel = { version = "*", features = ["sqlite"] }
-diesel_codegen = { version = "*", features = ["sqlite"] }
 r2d2-diesel = "*"
-r2d2 = "*"
+r2d2 = "0.8.*"
```

Comment:
Does that even work ?? :o

Reply:
It does work on my local dev, is it not supposes to?

Reply:
Nah I meant that I never saw it written this way, this is equivalent to "0.8" which is more common.